sentinelhub/data_collections.py

```diff
@@ -152,6 +152,7 @@ class DataCollectionDefinition:
     bands: Tuple[str, ...] = None
     collection_id: str = None
     is_timeless: bool = False
+    has_cloud_coverage: bool = False
```

Comment:
Hm, pylint is complaining about too many attributes -> [here](https://travis-ci.org/github/sentinel-hub/sentinelhub-py/jobs/761298763#L360).

I don't see a better solution but to suppress pylint for this class: `# pylint: disable=too-many-instance-attributes`

Reply:
Added.